pulsar-testclient/src/main/java/org/apache/pulsar/testclient/ManagedLedgerWriter.java

```diff
@@ -151,6 +156,12 @@ public static void main(String[] args) throws Exception {
             PerfClientUtils.exit(-1);
         }
 
+        if (arguments.metadataStoreUrl == null && arguments.zookeeperServers == null) {
```

Comment:
Does the param `arguments.zookeeperServers ` still be useful?

Reply:
Yes, we need to make it backward compatible.